language/transaction-builder/generator/src/lib.rs

```diff
@@ -23,7 +23,7 @@ pub mod csharp;
 /// Internals shared between languages.
 mod common;
 
-/// Read all ABI files in a directory.
+/// Read all ABI files in a directory. This supports both new and old `ScriptABI`s.
```

Comment:
@xli Do you need to call the new codegen tool on old ABI files in a near future? I suspect all the ABI files in the world are already migrated in this PR and we could just modify the ABI format in a breaking way (this one time).

Reply:
I don't need. But I don't think we need to do a breaking way. I'd like everything in a safe way. We can't predict what we may need in near future.

Reply:
I think it's safer to keep it this way at the moment. Otherwise, we would need to regenerate the ABIs for the legacy transaction scripts, which would actually be a decent amount of work to make sure that it's done properly and that we don't accidentally change the script bytecode.